libs/evcs-service/src/main/java/uk/gov/di/ipv/core/library/service/EvcsService.java

```diff
@@ -107,12 +115,13 @@ public Map<EvcsVCState, List<VerifiableCredential>> getVerifiableCredentialsBySt
     }
 
     @Tracing
-    public void storePendingVc(VerifiableCredential credential) throws EvcsServiceException {
+    public void storePendingVc(VerifiableCredential credential, EvcsVcProvenance provenance)
```

Comment:
Is this ever called with provenance other than `OFFLINE`? If you're worried about it being used inappropriately, we could call it `storePendingOfflineVc` or something.

Reply:
Done....Yeah thought so initially...as its only called from one place.